torch/onnx/_internal/fx/passes/decomp.py

```diff
@@ -32,6 +37,30 @@ def _rename_placeholder_targets(
     module.recompile()
 
 
+def wrapped_fn(fn, mode, allow_non_fake_inputs, args):
+    context = enable_python_dispatcher if mode == "symbolic" else contextlib.nullcontext
+    shape_env = ShapeEnv() if mode == "symbolic" else None
+    fake_mode = torch._dynamo.utils.fake_mode_from_tensors(args)
+    if mode is None:
+        fake_mode = FakeTensorMode(
+            shape_env=shape_env, allow_non_fake_inputs=allow_non_fake_inputs
+        )
+
+    args = [
+        (mode.from_tensor(t)
```

Comment:
@eellison onnx error from this line. `mode` is a str it appears?

Reply:
guess should be `fake_mode`?

Reply:
Thanks!

Reply:
@wschin for `export_without_parameters_and_buffers`

Reply:
I don't have clear idea. I need to see how the `input_` passed into that `assert`. It might be related to argument-wrapping code in exporter. If I get sometime this week, I will try debug.

Reply:
I am building this branch. Let's see if I can reproduce it tomorrow.